src/components/OtherFeaturesBlock/index.js

```diff
@@ -0,0 +1,3 @@
+import OtherFeaturesBlock from './OtherFeaturesBlock'
+
+export default OtherFeaturesBlock
```

Comment:
... and this file can be omitted altogether I think.

I'll skip these types of comments in other components.